libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -50,6 +50,19 @@ void StaticAnalyzer::endVisit(ContractDefinition const&)
 
 bool StaticAnalyzer::visit(FunctionDefinition const& _function)
 {
+	const bool isInterface = m_currentContract->contractKind() == ContractDefinition::ContractKind::Interface;
+	auto const& defaultVisibility = isInterface 
```

Comment:
Code formatting should be
```
auto const& defaultVisibility =
    isInterface ?
    "external" :
    Declaration::...;
```

Also please don't use `auto` in this case, I would not be sure if it is `string` or `char *`. Finally, since you use a literal string, I think `string` (without `const&`) would be better.

Oh and perhaps even just use `Declaration::visibilitiyToSTring(isInterface ? ...)`.